index.html

```diff
@@ -133,6 +133,9 @@ <h3>legend</h3>
                     if (data === "IETF") {
                       extension = ".gif"
                     }
+                    if (data === "Ecma") {
```

Comment:
Minor since it can't be both values
```suggestion
                    else if (data === "Ecma") {
```